src/index.js

```diff
@@ -1,10 +1,11 @@
 import dns from 'dns';
 import setConfig from './config';
-import Context from './context';
-import Callback from './callback';
 import Report from './report';
 import globals from './globals';
 
+let config = setConfig();
```

Comment:
I strongly dislike when things are "module-level" configured. I'm okay with being able to configure module-level _defaults_ though, as long as there is an override per object instance at the class constructor. The constructor would copy the module-level defaults into an instance var.

I might suggest exporting a setDefaults({}) to set the default config? Then anything passed to iopipe() would be an override of settings only used for that class instantiation.